drivers/remote/driver.go

```diff
@@ -64,7 +64,7 @@ func (d *driver) getCapabilities() (*driverapi.Capability, error) {
 		return nil, err
 	}
 
-	c := &driverapi.Capability{}
+	c := &driverapi.Capability{Multihost: capResp.Multihost}
```

Comment:
Same as previous comment. Just follow the same approach that we take as below for `Scope`. But add a backward compatibility `case "":` and set `c.ConnectivityScope = c.DataScope`